domains/oidc-mock/Mock/Pages/Connect/Signin.cshtml.cs

```diff
@@ -0,0 +1,68 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Oidc.Mock.Extensions;
+using Oidc.Mock.Models;
+
+namespace Oidc.Mock.Pages.Connect;
+
+public class SigninModel : PageModel
+{
+    private readonly ILogger<SigninModel> _logger;
+    private readonly Client _client;
+
+    public User[] Users { get; }
+
+    [FromForm]
+    public string? Name { get; set; }
+
+    [FromQuery(Name = "client_id")]
+    public string? ClientId { get; set; }
+
+    [FromQuery(Name = "redirect_uri")]
+    public string? RedirectUri { get; set; }
+
+    [FromQuery(Name = "state")]
+    public string? State { get; set; }
+
+    public SigninModel(User[] users, Client client, ILogger<SigninModel> logger)
+    {
+        _client = client;
+        _logger = logger;
+        Users = users;
+    }
+
+    public void OnGet()
+    {
+    }
+
+    public IActionResult OnPost()
+    {
+        _logger.LogDebug($"OnPost: ClientId={ClientId}, Name={Name}, RedirectUri={RedirectUri}");
+
+        var (isValid, validationError) = _client.Validate(ClientId, RedirectUri);
+        if (!isValid)
+        {
+            return BadRequest(validationError);
+        }
+
+        var userDescriptor = Users.FirstOrDefault(u => string.Equals(u.Name, Name, StringComparison.InvariantCultureIgnoreCase));
+        if (userDescriptor == null)
+        {
+            _logger.LogDebug($"OnPost: User '{Name}' not found");
```

Comment:
```suggestion
            _logger.LogError($"OnPost: User '{Name}' not found");
```
wouldn't thing of this as and debug but error